src/coreclr/vm/interpreter.cpp

```diff
@@ -882,6 +898,10 @@ CorJitResult Interpreter::GenerateInterpreterStub(CEEInfo* comp,
         // x8 through x15 are scratch registers on ARM64.
         IntReg x8 = IntReg(8);
         IntReg x9 = IntReg(9);
+#elif defined(HOST_LOONGARCH64)
```

Comment:
Does this section needed?

Reply:
> Does this section needed?

I argree with you.
Maybe the other CPUs are the same case.